### PR TITLE
P0 avatar alert

### PR DESCRIPTION
**Pull request by manojsarna**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,2 +1,364 @@
...
+
+html{
+    scroll-behavior: smooth;
+}
+
+ * {
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    /* font-family: 'Open Sans', sans-serif; */
```
Comment (iRohitGaur) [inline style.css]: remove these comments?

File: style.css [hunk 2]
```diff
@@ -1,2 +1,364 @@
...
+.alert-info{
+    
+    background-color: var( --COLOR-INFO-LIGHT);
+    color: var(--COLOR-INFO);
+    border: 1px solid var(--COLOR-INFO);
+    border-top-width: 4px;
+}
+
+
+
```
Comment (iRohitGaur) [inline style.css:152]: give consistent spacing between classes

File: style.css [hunk 3]
```diff
@@ -1,2 +1,364 @@
...
+/* ------ AVATAR -------- */
+
+.flex-row-wrap{
+    display: flex;
+    flex-wrap:wrap;
+    justify-content: center;
+    align-items: end;
+    gap: 2rem;
+}
+ /* if want to provide box-sizing:content-box */
```
Comment (iRohitGaur) [inline style.css]: ok?

**Review by iRohitGaur — APPROVED** [review]
Awesome work! cleanup required. Good to merge after

**Comment by PuneetP16** [comment]
You have given container element width 100vh instead of 100% and max-width 120rem in the body element which is causing the horizontal scroll bar to appear at the bottom. 
Solution: change container width to 100% and remove max-width from the body.
Really Awesome work on smashy hover avatars.

**Comment by m26unkwn** [comment]
hey, Manoj you can use a light green color for the tags 
image
and a grey color for text. it's too bright for the eyes.

**Comment by m26unkwn** [comment]
alert and avatar design is too good.

**Comment by deekshithmd** [comment]
looks good for me...

File: style.css [hunk 4]
```diff
@@ -1,2 +1,364 @@
...
+    flex-wrap:wrap;
+}
+
+.flex-column{
+    display: flex;
+    flex-direction: column;
+}
+
+.alert-box{
+    padding:2rem 2rem;
```
Comment (rahulyadav139) [inline style.css:135]: The second padding value is redundant (if it is similar to the previous one).

File: style.css [hunk 5]
```diff
@@ -1,2 +1,364 @@
...
+        /* FONT SIZE  */
+
+        font-size:62.5%;
+
+        /* COLORS */
+    
+        --COLOR-PRIMARY:#44d62c;
+        --COLOR-LIGHT-PRIMARY:#73e161;
+        --COLOR-DARK-PRIMARY:#1dd800;
+
```
Comment (rahulyadav139) [inline style.css]: Awesome. The root is loaded with CSS variables to ease out. Codes are reliable and arranged in nature.

File: index.html [hunk 6]
```diff
@@ -9,7 +9,305 @@
...
+
+                <div class="component-show-container flex-column-wrap">
+                    
+                    <!--  INFO ALERT -->
+                    <div class="alert-box alert-info">
+                        <span class="alert-icon">
+                            <i class="fas fa-info-circle"></i>
+                        </span>
+                        <span>An Info Alert Example with text link 
+                        <a class="link-text link-text-info">Please Check!</a>
```
Comment (gauravsinhaweb) [inline index.html:44]: I have opted few minor points that can be improved:
1. The <a> tag in HTML is used to create a hyperlink on the webpage, you can use `href` in <a> tag for good coding practices. 
2. The Avatar image size is too large, which can affect the performance of the website. You can reduce the image height width to whatever it requires.

overall Nice work done, LGTM to merge changes.

**Comment by Ramandeep9898** [comment]
looks gud to me:) love the colors

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

**Comment by j24m** [comment]
Looks clean, the color combination looks beautiful.

**Comment by veereshbv04** [comment]
There is horizontal scroll bar comming in deploy preview, I guess this can be corrected now

**Pull request merged**